lang/cs/Org.Apache.REEF.IMRU/API/IMRUJobDefinitionBuilder.cs

```diff
@@ -72,7 +72,7 @@ public IMRUJobDefinitionBuilder()
             _updateTaskMemory = 512;
             _coresPerMapper = 1;
             _updateTaskCores = 1;
-            _maxRetryNumberInRecovery = 0;
+            _maxRetryNumberInRecovery = -1;
```

Comment:
0 and -1 seem to produce the same behavior in `IMRUDriver` (no retries). But -1 seems weird in context of max number of something. Is there a specific reason to use -1 instead of 0 here?